src/topp/IDFileConverter.cpp

```diff
@@ -170,6 +176,9 @@ class TOPPIDFileConverter :
     registerFlag_("ignore_proteins_per_peptide", "[Sequest only] Workaround to deal with .out files that contain e.g. \"+1\" in references column,\n"
                                                  "but do not list extra references in subsequent lines (try -debug 3 or 4)", true);
     registerStringOption_("scan_regex", "<expression>", "", "[Mascot, pepXML, Percolator only] Regular expression used to extract the scan number or retention time. See documentation for details.", false, true);
+    registerFlag_("add_spectrum_references", "Adds the spectrum files spectrum ID reference to each PeptideIdentification. Warning: works only with mz_file set", true);
```

Comment:
Better: "Adds a reference to the spectrum file's spectrum ID to each peptide identification. This requires 'mz_file' to be set."
"PeptideIdentification" is the name of the OpenMS class, which is irrelevant for users. The actual entity is a "peptide identification"!